fedmsg/core.py

```diff
@@ -62,7 +62,7 @@ def __init__(self, **config):
         self.log = logging.getLogger("fedmsg")
 
         self.c = config
-        self.hostname = socket.gethostname().split('.', 1)[0]
+        self.hostname = socket.gethostname().split('.', maxsplit=1)[0]
```

Comment:
``maxsplit`` is a positional argument in Python 2.7, which we unfortunately still have to support.